tests/unit/test_charm.py

```diff
@@ -258,6 +276,19 @@ def test_given_srsenb_service_is_running_when_on_stop_then_folders_content_is_re
             ]
         )
 
+    @patch("charm.SrsLteCharm._get_current_status")
```

Comment:
Don't write tests that depend on this given method. Here we don't want to validate that the method was called. What we would want to validate for example is that when the config is bad we get Blocked.

Reply:
Fixed.